cookbooks/fb_logrotate/attributes/default.rb

```diff
@@ -103,6 +103,7 @@
   'systemd_timer' => systemd_timer,
   'systemd_settings' => {
     'OnCalendar' => 'daily',
+    'Persistent' => 'false',
```

Comment:
Oh, sorry I missed this the first time. Please name this a real Boolean.

Reply:
@jaymzh Fixed! https://github.com/facebook/chef-cookbooks/pull/326/commits/bb1d4c0dc18d65f8e860ad705fb4c1b66062e69d